viennacl/src/main/scala/org/apache/mahout/viennacl/opencl/javacpp/DenseColumnMatrix.scala

```diff
@@ -36,12 +36,24 @@ import org.bytedeco.javacpp.annotation._
 @Name(Array("viennacl::matrix<double,viennacl::column_major>"))
 final class DenseColumnMatrix(initDefault:Boolean = true) extends MatrixBase {
 
-  def this(nrow: Int, ncol: Int, ctx: Context = new Context()) {
+  def this(nrow: Int, ncol: Int) {
+    this(false)
+    allocate(nrow, ncol, new Context())
+  }
+
+  def this(nrow: Int, ncol: Int, ctx: Context) {
     this(false)
     allocate(nrow, ncol, ctx)
   }
 
-  def this(data: DoublePointer, nrow: Int, ncol: Int, ctx: Context = new Context(Context.MAIN_MEMORY)) {
```

Comment:
hmm. when running `mvn test` on a scala 2.11.2 build it seems to bee looking for ViennCL built for 2.10:  `mahout-native-viennacl_2.10 `
```
[INFO] Copying 3 resources
[INFO] 
[INFO] --- scala-maven-plugin:3.2.0:testCompile (scala-test-compile) @ mahout-native-viennacl_2.10 ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-compiler-plugin:3.3:testCompile (default-testCompile) @ mahout-native-viennacl_2.10 ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:2.18.1:test (default-test) @ mahout-native-viennacl_2.10 ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- scalatest-maven-plugin:1.0:test (test) @ mahout-native-viennacl_2.10 ---
```
Maybe the scalatest plugin in scala version is hardcoded in a pom somewere?.